pkg/helpers/config/userdata.yaml

```diff
@@ -7,7 +7,8 @@ runcmd:
   - sudo service docker start
   - echo "${USERDATA_BEGIN}" >> /var/log/userdata-output
   - sudo docker pull ${VALIDATOR_IMAGE}
+  - echo "${CACERT}" | base64 --decode > /proxy.pem
   # Use `|| true` to ignore failure exit codes, we want the script to continue either way
-  - sudo docker run --env "AWS_REGION=${AWS_REGION}" -e "START_VERIFIER=${VALIDATOR_START_VERIFIER}" -e "END_VERIFIER=${VALIDATOR_END_VERIFIER}" ${VALIDATOR_IMAGE} --timeout=${TIMEOUT}  >> /var/log/userdata-output || echo "Failed to successfully run the docker container"
+  - sudo docker run -v /proxy.pem:/proxy.pem -e "HTTP_PROXY=${HTTP_PROXY}" -e "HTTPS_PROXY=${HTTPS_PROXY}" --env "AWS_REGION=${AWS_REGION}" -e "START_VERIFIER=${VALIDATOR_START_VERIFIER}" -e "END_VERIFIER=${VALIDATOR_END_VERIFIER}" ${VALIDATOR_IMAGE} --timeout=${TIMEOUT} --cacert=/proxy.pem --no-tls=${NOTLS}  >> /var/log/userdata-output || echo "Failed to successfully run the docker container"
```

Comment:
What happens if these aren't provided?

Reply:
I think there's idempotence, nothing happens if no provided, but if you have suggestions on how we can communicate it, I can make the change upon that way

Reply:
I looked at it today, probably I need to refactor here(now or future), as I pass the empty proxy file when `${CACERT}` equals to "".
Although it doesn't break any functionality(when there's no ca/https proxy), it'd be nice to refactor here to a form `proxy.pem` was being passed based on the parameter is provided

Reply:
this is now separated based on the CACERT flag has been passed